### ISSUE TITLE
Unclear on how to apply force or manipulate the physics body.

### ISSUE THREAD

**Issue by agmcleod** [WhitestormJS/physics-module-ammonext#2]
Trying out a prototype idea with whitestorm. Was able to setup a scene, but running into issues trying to do things with the physics engine. I see a number of useful methods here: https://github.com/WhitestormJS/physics-module-ammonext/blob/master/src/modules/physicsPrototype.js, but i have no way to execute them. I've tried accessing:

`object.modules[0]`, which is the instance of `PHYSICS.BoxModule` i'm using, but it has none of these methods.

**Comment by hirako2000** [comment]
@agmcleod, checked the [codepen](https://codepen.io/sasha240100/pen/JELBGX)? also check the [examples](https://whs-dev.surge.sh/examples/?basic/cube)
Come in [discord](https://discordapp.com/channels/238405369859145729/238405369859145729) if you have more troubles.

**Comment by agmcleod** [comment]
Yep, was referring to that codepen for getting started, though it doesn't have any examples on applying force or anything to move an object around. Tried looking at the examples, but I can't seem to find the source for them.

**Comment by hirako2000** [comment]
Check the physics module [page](https://github.com/WhitestormJS/physics-module-ammonext)
You can access all the examples sources [here](https://github.com/WhitestormJS/whitestorm.js/tree/beta/examples)

I suggest we talk on [discord](https://discordapp.com/channels/238405369859145729/238405369859145729), @sasha240100  is far more familiar with the physics module than I. he will certainly help to get what you need to achieve.
